model/restaurantModel.go

```diff
@@ -0,0 +1,21 @@
+package model
```

Comment:
can we put in a sub pkg either `view` or `db` or something? to differentiate from the models we will have for our api(s)